docs/docs/getting-started/install.md

```diff
@@ -111,6 +111,14 @@ If you don't have a cloud subscription, you can try [MiniConstellation](first-st
 <tabs groupId="csp">
 <tabItem value="azure" label="Azure">
 
+The following resource providers need to be [registered](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/resource-providers-and-types#register-resource-provider) in your subscription:
+* `Microsoft.Compute`
+* `Microsoft.ManagedIdentity`
+* `Microsoft.Network`
+* `microsoft.insights`
```

Comment:
That's actually spelled lowercase by Microsoft.

Reply:
The docs page you linked contains a link to https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/azure-services-resource-providers where it's spelled uppercase. However, if in reality/technically it's spelled lowercase, it might be safer to keep it lowercase.

Reply:
Good thing they themselves mess this up. This is what it looks like on the portal:
![Screenshot 2023-03-01 at 12 32 33](https://user-images.githubusercontent.com/26608194/222127721-c67d8796-177c-46d2-8bd8-b2689a346e46.png)

The API client also returns "microsoft.insights", but at least the Azure CLI seems to accept the uppercase one?
No idea, it's a bit of the mess. Lowercase technically is safer.